cfme/tests/cloud/test_tag_mapping.py

```diff
@@ -262,5 +243,21 @@ def test_ec2_tags_images():
             3. Go to summary of this image  and check whether there is
             test:testing in Labels field
             4. Delete that tag
+        testSteps:
```

Comment:
`testSteps` are repeated twice? Did you mean to delete L241-L245?

Reply:
I merged 2 manual tests to one automated.. I did not know what to do with manual steps

Reply:
It was two manual tests, that I have merged in one automated. I didn't know what to do with manual steps. Now I have merged manual steps too.